modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -128,10 +128,10 @@ include::howtos:example$provisioning_resources_collections.py[tag=drop_collectio
 include::howtos:example$provisioning_resources_collections.py[tag=drop_scope]
 ----
 
-Note that the most minimal permissions to create and drop a Scope or Collection is
-https://docs.couchbase.com/server/7.0/learn/security/roles.html#manage-collections-in-scope[Scope Admin]
+Note that the most minimal permissions to create and drop a Scope or Collections is
```

Comment:
ah sorry, missed this, `Collection` instead of `Collections`.

Reply:
Hah, I was sure you'd pulled me up on this already, and indeed it is in the dotnet version I was using as template, so correcting there too...

Reply:
Ah, reminds me I should actually look at the .NET PR too, I'd totally forgotten sorry.